<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-EAC6D0I" name="com.TestCases.TC_002_Post_ValidateHomePage" tests="3" failures="0" timestamp="13 Jan 2021 09:21:41 GMT" time="0.038" errors="1">
  <testcase name="ValidateLoginPageStatuscode" time="0.009" classname="com.TestCases.TC_002_Post_ValidateHomePage"/>
  <testcase name="ValidateloginPageStatusLine" time="0.006" classname="com.TestCases.TC_002_Post_ValidateHomePage"/>
  <testcase name="ValidateLoginResponseBody" time="0.023" classname="com.TestCases.TC_002_Post_ValidateHomePage">
    <error type="java.lang.ClassCastException" message="io.restassured.internal.RestAssuredResponseImpl cannot be cast to io.restassured.path.json.JsonPath">
      <![CDATA[java.lang.ClassCastException: io.restassured.internal.RestAssuredResponseImpl cannot be cast to io.restassured.path.json.JsonPath
at com.TestCases.TC_002_Post_ValidateHomePage.ValidateLoginResponseBody(TC_002_Post_ValidateHomePage.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- ValidateLoginResponseBody -->
</testsuite> <!-- com.TestCases.TC_002_Post_ValidateHomePage -->
